src/planwise/client/dialog.cljs

```diff
@@ -1,11 +1,12 @@
 (ns planwise.client.dialog
   (:require [planwise.client.ui.rmwc :as m]
+            [re-frame.core :refer [dispatch]]
             [planwise.client.utils :as utils]))
 
 (defn new-dialog
-  [{:keys [open? title content accept-fn cancel-fn delete-fn]}]
+  [{:keys [open? title content delete-dispatch accept-fn cancel-fn delete-fn]}]
   [m/Dialog {:open open?
-             :on-accept accept-fn
+             :on-accept (or  accept-fn #(dispatch [:dialog/delay-dispatch delete-dispatch]))
```

Comment:
I don't like mix and matching callback style attributes with dispatch vectors. For generality, I'd prefer to just use `accept-fn` and make the dispatch function at the use point.